netstandard/tools/targets/NETStandard.Library.Trimming.targets

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="TrimFiles" AssemblyFile="$(NETStandardToolsTaskDirectory)NETStandard.Tools.dll" />
+
+  <Target Name="_determineTrimPackageInputs">
```

Comment:
What made you name targets with an underscore now? That is like naming a method with and underscore... yuck.